android/src/facebook/TiFacebookModule.java

```diff
@@ -115,6 +115,20 @@ public static void onAppCreate(TiApplication app)
 	public static TiFacebookModule getFacebookModule() {
 		return module;
 	}
+    
+    @Override
+    public void onResume(Activity activity) {
+        super.onResume(activity);
+        Log.d(TAG, "Calling activateApp");
+        AppEventsLogger.activateApp(activity);
```

Comment:
However I am not quite sure what is meant by "someone is not using AppEventsLogger", as it is a static method of the class (no new logger instance is required), and the class was already imported before this update, just not configured